README.md

```diff
@@ -273,6 +275,14 @@ Then run llrt:
 
     make run
 
+## Benchmark Methodology
+Although Init Duration [reported by Lambda](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtime-environment.html) is commonly used to understand cold start impact on overall request latency, this metric does not include the time needed to copy code into the Lambda sandbox. 
+
+The technical definition of Init Duration ([source](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-logging.html#node-logging-output)):
```

Comment:
Also clarify that we're measuring init + duration for our Lambda results. This might be interpreted as the benchmark only shows `init`, for the Lambda column

Reply:
Done!